src/lib/components/SideBar/SideBarLink.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+  import { getDrawerStore } from "@skeletonlabs/skeleton";
+
+  /**
+   * Should be an SVG icon from Iconify `~icons/material-symbols`
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  export let Icon: any;
+  export let label: string;
+  export let href: string;
+
+  const closeDrawer = getDrawerStore().close;
+</script>
+
+<a id="sidebar-link-{label}" {href} on:click={closeDrawer}>
+  <div
+    class="flex flex-row whitespace-nowrap hover:bg-secondary-500 w-full gap-4 p-5 lg:p-3 rounded-lg"
+  >
+    <Icon class="w-14 h-14 lg:w-10 lg:h-10 ml-3" />
+    <span class="text-2xl lg:text-xl mt-auto mb-auto">{label}</span>
+  </div>
```

Comment:
```suggestion
  <div
    class="flex flex-row items-center whitespace-nowrap hover:bg-secondary-500 w-full gap-4 p-5 lg:p-3 rounded-lg"
  >
    <Icon class="w-14 h-14 lg:w-10 lg:h-10 ml-3" />
    <span class="text-2xl lg:text-xl">{label}</span>
  </div>
```

---------
Also, if the larger sizes were intended to be for larger screens, then the `lg:` selector should've been applied to them instead:

`    <Icon class="w-10 h-10 lg:w-14 lg:h-14 ml-3" />`, for example

![chrome-capture-2023-11-26](https://github.com/icssc/ZotMeet/assets/100006999/470500ac-e44e-4e6f-abe3-83752c6fd1d8)


Otherwise, disregard this!

Reply:
I meant for the font to be larger in the drawer, which is enabled at the `md` threshold, not `lg`.